### PR TITLE
Feature/dashboard charts reorganization

### PR DESCRIPTION
**Pull request by frenchbread**

### CODE REVIEW

**Comment by frenchbread** [comment]
@apinf/developers Please review and test.

File: client/views/dashboard/charts/charts.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#line-chart, #overview-chart{
+#move-chart, #overview-chart{
     width:100%
```
Comment (brylie) [inline client/views/dashboard/charts/charts.css]: Use whitespace after colon, for readability.

File: client/views/dashboard/charts/charts.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 }
-#line-chart-box, #overview-chart-box, #data-table-box, #filter-box, #map-box{
-    padding:10px;
+#overview-chart-box, #move-chart-box, #data-table-box, #filter-box, #map-box{
+    padding:5px;
```
Comment (brylie) [inline client/views/dashboard/charts/charts.css]: Use whitespace after colon; use ems or rems instead of px

**Comment by brylie** [comment]
This looks good. I will merge it, but in the future be more careful to add whitespace for readability and use relative units (ems or rems) for layout.

**Pull request merged**